modules/querying-operator-status-after-installation.adoc

```diff
@@ -21,7 +21,57 @@ You can check Operator status at the end of an installation. Retrieve diagnostic
 $ oc get clusteroperators
 ----
 
-. If any Operators fail to become available, view Operator events:
+. Verify that all of the required certificate signing requests (CSRs) are approved. Some cluster Operators might not become available if there are pending CSRs.
```

Comment:
csr issues will lead to some unready nodes, or not found some expected nodes, I do not think they should be linked clusteroperators troubleshooting.

Reply:
@jianlinliu in my test UPI installation for OCP on AWS, the Console Operator and others did not become active until I approved pending CSRs. This prevented me from being able to see the OCP web console. I have included this here because it would be useful for the reader to check that CSRs are approved in this type of scenario.

Please let me know if this makes sense.

Reply:
From my understanding, not approved csr -> node not ready -> some cluster operator does not get ready (not only console operator). So before check cluster operators, firstly need ensure all nodes get ready.

Reply:
Thank you for clarifying that. Since this module is specifically about troubleshooting operator issues after an installation, the CSR approval steps are still relevant here. From your comment my understanding is still that some cluster operators do not become ready if some CSRs are not approved. I have updated the step to mention that nodes might not be ready also, for context:

". Verify that all of the required certificate signing requests (CSRs) are approved. Some nodes might not move to a `Ready` status and some cluster Operators might not become available if there are pending CSRs."

Does this meet your requirement?